pkg/util/ctxgroup/ctxgroup_test.go

```diff
@@ -42,3 +43,17 @@ func TestErrorAfterCancel(t *testing.T) {
 		}
 	})
 }
+
+func TestPanic(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+
+	panics := func(_ context.Context) error {
+		panic(1)
+	}
+
+	require.Panics(t, func() {
```

Comment:
Can you assert that the outer panic contains the inner panic's stacktrace?

Reply:
Done.